config/form-content/mandataris-ext/form.ttl

```diff
@@ -0,0 +1,138 @@
+@prefix mu: <http://mu.semte.ch/vocabularies/core/>.
+@prefix displayTypes: <http://lblod.data.gift/display-types/>.
+@prefix schema: <http://schema.org/>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix nie: <http://www.semanticdesktop.org/ontologies/2007/01/19/nie#>.
+@prefix foaf: <http://xmlns.com/foaf/0.1/>.
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/>.
+@prefix mandaat: <http://data.vlaanderen.be/ns/mandaat#>.
+@prefix org: <http://www.w3.org/ns/org#>.
+@prefix extlmb: <http://mu.semte.ch/vocabularies/ext/lmb/>.
+
+ext:persoonF
+    a form:Field;
+    form:displayType displayTypes:personSelector;
+    sh:group ext:mandatarisPG;
+    sh:name "Persoon";
+    sh:order 2;
+    sh:path mandaat:isBestuurlijkeAliasVan.
+ext:mandaatF
+    a form:Field;
+    form:displayType displayTypes:mandatarisMandaatSelector;
+    sh:group ext:mandatarisPG;
+    sh:name "Mandaat";
+    sh:order 3;
+    sh:path org:holds.
+ext:mandatarisStatusCodeF
+    a form:Field;
+    form:displayType displayTypes:mandatarisStatusCodeSelector;
+    sh:group ext:mandatarisPG;
+    sh:name "Status";
+    sh:order 4;
+    sh:path mandaat:status;
+    form:options """{"conceptScheme":"http://data.vlaanderen.be/id/conceptscheme/MandatarisStatusCode","searchEnabled":true}""";
+    form:validatedBy 
+        [
+            a form:RequiredConstraint;
+            form:grouping form:Bag;
+            sh:path mandaat:status;
+            sh:resultMessage "Status is verplicht."
+    ].
+ext:rangordeF
+    a form:Field;
+    form:displayType displayTypes:mandatarisRangorde;
+    sh:group ext:mandatarisPG;
+    sh:name "Rangorde";
+    sh:order 6;
+    sh:path mandaat:rangorde;
+    form:validatedBy [
+      a ext:ValidRangorde;
+      form:grouping form:MatchSome;
+      sh:severity sh:Warning;
+      sh:resultMessage "Er werd geen geldige rangorde herkend";
+      sh:path mandaat:rangorde
+    ].
+ext:startF
+    a form:Field;
+    form:displayType displayTypes:dateTime;
+    sh:group ext:mandatarisPG;
+    sh:name "Start";
+    sh:order 7;
+    sh:path mandaat:start;
+    form:validatedBy 
+        [
+            a form:RequiredConstraint;
+            form:grouping form:Bag;
+            sh:path mandaat:start;
+            sh:resultMessage "Startdatum is verplicht."
+    ].
+ext:eindeF
+    a form:Field;
+    form:displayType displayTypes:dateTime;
+    sh:group ext:mandatarisPG;
+    sh:name "Einde";
+    sh:order 8;
+    sh:path mandaat:einde.
+ext:fractieF
+    a form:Field;
+    form:displayType displayTypes:mandatarisFractieSelector;
+    sh:group ext:mandatarisPG;
+    sh:name "Fractie";
+    sh:order 9;
+    sh:path ( org:hasMembership org:organisation );
+    form:validatedBy 
+        [
+            a form:RequiredConstraint;
+            form:grouping form:Bag;
+            sh:path ( org:hasMembership org:organisation );
+            sh:resultMessage "Dit veld is verplicht."
+    ].
+ext:beleidsdomeinCodeF
```

Comment:
Rangorde shows as expected but the beleidsdomeinen is not shown in the form. When should this be triggered?